primefaces/src/main/frontend/packages/idlemonitor/idlemonitor/src/idlemonitor.widget.js

```diff
@@ -44,14 +44,18 @@ PrimeFaces.widget.IdleMonitor = class IdleMonitor extends PrimeFaces.widget.Base
                 $this.cfg.onidle.call($this);
             }
 
-            $this.callBehavior('idle');
+            if($this.hasBehavior('idle')) {
```

Comment:
IMO callBehavior should check "hasBehavior" by itself

i introduced hasBehavior for other cases, were a manual AJAX request needs to be fired

Reply:
ahh, i see
maybe we should add a param "fallbackToDomEvent" with default true, and set this to false here

Reply:
OK i thought about that i like that idea!